docs/en/notes.md

```diff
@@ -0,0 +1,19 @@
+# Developer documentation
+
+## Integration notes
+
+### Filters use the CKAN SQL API endpoint
+
+The filter components in the frontend example app uses the [`datastore_search_sql`](https://docs.ckan.org/en/2.8/maintaining/datastore.html#ckanext.datastore.logic.action.datastore_search_sql)
+search endpoint on the target CKAN instance. It's important to note that while data.govt.nz should have this optional
+CKAN extension enabled for all datasets, not every CKAN instance will by default.
```

Comment:
```suggestion
CKAN extension enabled for all datasets, not every CKAN instance may have this functionality enabled.
```
I only suggest this because Cam let us know that it does default to "on"